dom-event-delegation/main.js

```diff
@@ -0,0 +1,12 @@
+function handleClick(event) {
+  console.log('value of event.target :', event.target);
+  console.log('value of event.target.tagName:', event.target.tagName);
+  if (event.target.matches('button') && event.target.tagName === 'BUTTON') {
```

Comment:
You don;t need both of these conditions, you can use one or the other and it will still work 🤓

Reply:
Got it, thanks!